src/solvers/eigsolve.jl

```diff
@@ -0,0 +1,38 @@
+
+function eigsolve_updater(
+  init;
+  psi_ref!,
+  PH_ref!,
+  outputlevel,
+  which_sweep,
+  region_updates,
+  which_region_update,
+  region_kwargs,
+  updater_kwargs,
+)
+  default_updater_kwargs = (;
+    solver_which_eigenvalue=:SR,
+    ishermitian=true,
+    tol=1e-14,
+    krylovdim=3,
+    maxiter=1,
+    outputlevel=0,
+    eager=false,
+  )
+  updater_kwargs = merge(default_updater_kwargs, updater_kwargs)  #last collection has precedence
+  howmany = 1
+  which = updater_kwargs[:solver_which_eigenvalue]
+  vals, vecs, info = KrylovKit.eigsolve(
+    PH_ref![],
+    init,
+    howmany,
+    which;
+    ishermitian=updater_kwargs[:ishermitian],
+    tol=updater_kwargs[:tol],
+    krylovdim=updater_kwargs[:krylovdim],
+    maxiter=updater_kwargs[:maxiter],
+    verbosity=updater_kwargs[:outputlevel],
+    eager=updater_kwargs[:eager],
+  )
+  return vecs[1], (; info, energies=vals)
```

Comment:
```suggestion
  return vecs[1], (; info, eigvals=vals)
```
since they won't always be energies (that would depend on the operator).